flatland_viz/include/flatland_viz/flatland_window.h

```diff
@@ -44,24 +44,77 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
+// namespace rviz;
+
 #include <ros/ros.h>
 #include <QLabel>
 #include <QMainWindow>
+#include <QPushButton>
 #include <QWidget>
 #include "flatland_viz/flatland_viz.h"
 
+//#include "rviz/display_context.h"
+//#include "rviz/tool.h"
+//#include "rviz/tool_manager.h"
+// class ToolManager;
+// namespace rviz {
+// class ToolManager;
+// }
+#include "rviz/display.h"
+#include "rviz/display_context.h"
+#include "rviz/displays_panel.h"
+#include "rviz/env_config.h"
+#include "rviz/failed_panel.h"
+#include "rviz/help_panel.h"
+#include "rviz/load_resource.h"
+#include "rviz/loading_dialog.h"
+#include "rviz/new_object_dialog.h"
+#include "rviz/panel_dock_widget.h"
+#include "rviz/panel_factory.h"
+#include "rviz/properties/status_property.h"
+#include "rviz/render_panel.h"
+#include "rviz/screenshot_dialog.h"
+#include "rviz/selection/selection_manager.h"
+#include "rviz/selection_panel.h"
+#include "rviz/splash_screen.h"
+#include "rviz/time_panel.h"
+#include "rviz/tool.h"
+#include "rviz/tool_manager.h"
+#include "rviz/tool_properties_panel.h"
+#include "rviz/view_manager.h"
+#include "rviz/views_panel.h"
+#include "rviz/visualization_frame.h"
+#include "rviz/visualization_manager.h"
+#include "rviz/widget_geometry_change_detector.h"
+#include "rviz/yaml_config_reader.h"
+#include "rviz/yaml_config_writer.h"
+
 class FlatlandWindow : public QMainWindow {
   Q_OBJECT
  public:
   FlatlandWindow(QWidget* parent = 0);
+  // rviz::ToolManager* tool_manager_;
+  // here
+  // rviz::VisualizationFrame* vFrame_;
```

Comment:
more test/debug commented out code to clean up

Reply:
remove here